packages/aggregator/src/analytics/aggregateAnalytics/aggregations/getFinalValuePerPeriod.js

```diff
@@ -120,10 +126,13 @@ class FinalValueAggregator {
           mostRecentValue = valueForPeriod;
         }
         if (mostRecentValue !== undefined) {
-          values.push({
+          const valueForOneOrgUnit = {
             ...mostRecentValue,
             period,
-          });
+          };
+          if (!checkIfFillingContinuousValues && !valueForPeriod)
+            valueForOneOrgUnit.value = fillEmptyPeriodsWith;
```

Comment:
```suggestion
          if (!checkIfFillingContinuousValues && !valueForPeriod) {
            valueForOneOrgUnit.value = fillEmptyPeriodsWith;
          }
```

Minor thing, but I think we always use brackets around code blocks in our codebase 👍